addons/fix-uploaded-svgs/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Fix broken SVG uploads",
-  "description": "Fixes a bug in the Scratch editor that fails to properly upload SVG files created with some image editors, such as Affinity Designer.",
+  "name": "Fixed SVG uploads",
```

Comment:
Better words for this, not "fixed"?

Reply:
Yeah, I don't like any of the 2 changes here.

Reply:
SVG uploads fix, how about that?